### PR TITLE
update(manifests): Make the dev manifests look more like the deis chart

### PR DESCRIPTION
**Pull request by jchauncey**
There are things in the workflow manifests that are not in the deis chart. If there is something missing in these manifests we need to add just let me know.

### CODE REVIEW

File: manifests/deis-workflow-rc.yml [hunk 1]
```diff
@@ -16,7 +16,7 @@ spec:
     spec:
       containers:
         - name: deis-workflow
-          image: quay.io/deisci/workflow:v2-beta
+          image: quay.io/deis/workflow:2.0.0-alpha
```
Comment (helgi) [inline manifests/deis-workflow-rc.yml]: master is `beta`

File: manifests/deis-workflow-service.yml [hunk 2]
```diff
@@ -2,13 +2,21 @@ apiVersion: v1
 kind: Service
 metadata:
   name: deis-workflow
+  namespace: deis
   labels:
     heritage: deis
+    routable: "true"
```
Comment (helgi) [inline manifests/deis-workflow-service.yml]: see #235 - Landing that one will simplify this PR

File: manifests/deis-workflow-service.yml [hunk 3]
```diff
@@ -2,13 +2,21 @@ apiVersion: v1
...
+  annotations:
+    deis.io/routerConfig: |
+      {
+        "domains": [ "deis" ],
+        "connectTimeout": 10,
+        "tcpTimeout": 1200
+      }
```
Comment (helgi) [inline manifests/deis-workflow-service.yml]: This format is going away so lets hold off until that has all landed

**Comment by helgi** [comment]
You'll want to rebase this one since the manifest changed recently due to the new annotation format

**Comment by jchauncey** [comment]
Im just going to close this for now. I jumped the gun on this.

**Pull request closed without merging**